sentry_sdk/tracing_utils.py

```diff
@@ -896,6 +830,88 @@ def _sample_rand_range(parent_sampled, sample_rate):
         return sample_rate, 1.0
 
 
+def create_span_decorator(op=None, name=None, attributes=None):
+    # type: (Optional[str], Optional[str], Optional[dict[str, Any]]) -> Any
+    """
+    Create a span decorator that can wrap both sync and async functions.
+
+    :param op: The operation type for the span.
+    :param name: The name of the span.
+    :param attributes: Additional attributes to set on the span.
```

Comment:
Do we intend to add more parameters in the future? How were these ones selected?